packages/lrod/PKGBUILD

```
@@ -0,0 +1,50 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=lrod
+pkgver=Version.r32.g57d9cab
+pkgrel=1
+pkgdesc='Perform subdomain enumeration, endpoint recognition, and more.'
+arch=('any')
+groups=('blackarch' 'blackarch-recon')
+url='https://gitlab.com/Edu0x01/Lrod'
+license=('MIT')
+depends=('python' 'python-virtualenv')
+makedepends=('git' 'python-setuptools' 'python-pip')
```

```suggestion
makedepends=('git' 'python-pip')
```

not needed, there is no package